src/components/MentorsList/MentorsList.js

```diff
@@ -30,6 +37,7 @@ const MentorsList = () => {
       <Cards
         mentors={mentors}
         favorites={favorites}
+        onAvatarClick={onAvatarClick}
```

Comment:
I love that idea and implemented it. Let me know what you think.
(I do not love the `<a>` inside `Link` though and seems like [I'm not the only one](https://github.com/vercel/next.js/discussions/32233).
Since we can't really fix it, we can "hide" the ugliness behind our own `Link` component, what do you think?

Reply:
I feel your pain.  That requirement has been annoying since day 1, but as you've pointed out it can be hidden away with our own `<Link>`.  I'm _very_ interested in getting something like that added to the system.  As [you can see](https://github.com/vercel/next.js/blob/canary/packages/next/client/link.tsx#L224), Next is just wrapping the `string` type children in an `<a>` tag for you, so it's not a big deal to wrap all children in an `<a>` ourselves.

Reply:
Just added. If it's over engineering, please let me know 😅